typegate/tests/runtimes/python_wasi/python_wasi_test.ts

```diff
@@ -241,3 +241,57 @@ Meta.test("Python WASI: infinite loop or similar", async (t) => {
   //     .on(e);
   // });
 }, { sanitizeOps: false });
+
+Meta.test("Python WASI: typegate reloading", async (metaTest) => {
+  const load = async () => {
+    const e = await metaTest.engine("runtimes/python_wasi/python_wasi.ts");
+    return e;
```

Comment:
I think you should call or return `await metaTest.engine(".....")` directly, the last line feels unnecessary.

Reply:
Yeah.. I guess that's cleaner. will address it.